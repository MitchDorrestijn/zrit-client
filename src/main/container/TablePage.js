import React, { Component } from 'react';
import Header from '../components/Header';
import TextContent from '../components/TextContent';
import Footer from '../components/Footer';
import { Container, Row, Col } from 'reactstrap';

export default class TablePage extends Component {
  render(){
    return(
      <Container>
        <Row>
          <Col>
            <Header />
            <TextContent title="Betalingen">
              Cookie tiramisu bonbon dragée danish icing danish I love. Lemon drops halvah tiramisu tootsie roll cheesecake. Candy canes candy canes jelly beans I love. Tootsie roll I love cheesecake gingerbread dragée cupcake dragée marzipan cupcake.
            </TextContent>
            [tabel komt hier]
            <Footer />
          </Col>
        </Row>
      </Container>
    );
  }
}
