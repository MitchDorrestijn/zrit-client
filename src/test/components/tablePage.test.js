import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import TablePage from '../../main/container/TablePage';

const wrapper = shallow(<TablePage />);

test('TablePage component should match snapshot', () => {
  expect(toJSON(wrapper)).toMatchSnapshot();
});

test('Should render one Container component', () => {
  expect(wrapper.find('Container').length).toBe(1);
});

test('Should render one Row component', () => {
  expect(wrapper.find('Row').length).toBe(1);
});

test('Should render one Col component', () => {
  expect(wrapper.find('Col').length).toBe(1);
});

test('Should render one Footer component', () => {
  expect(wrapper.find('Footer').length).toBe(1);
});

test('Should render one Header component', () => {
  expect(wrapper.find('Header').length).toBe(1);
});
