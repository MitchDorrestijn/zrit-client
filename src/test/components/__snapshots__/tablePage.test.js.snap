// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TablePage component should match snapshot 1`] = `
<Container
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Header />
      <TextContent
        title="Betalingen"
      >
        Cookie tiramisu bonbon dragée danish icing danish I love. Lemon drops halvah tiramisu tootsie roll cheesecake. Candy canes candy canes jelly beans I love. Tootsie roll I love cheesecake gingerbread dragée cupcake dragée marzipan cupcake.
      </TextContent>
      [tabel komt hier]
      <Footer />
    </Col>
  </Row>
</Container>
`;
